src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,49 @@
+.nav {
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    list-style: none;
+    position: relative;
+    margin-left: 20px;
```

Comment:
Consider using relative units (like percentages or ems) for margin to improve responsiveness.